adhocteam/Head-Start-TTADP
# [TTAHUB-168] add dashboard overview statistics widget

## Description of change
Add overview statistics to the dashboard page

- This is a large PR so I've added comments annotating my changes.
- I'm also trying to merge with the "create-regional-dashboard-page" branch, it seems (to me) like it make sense to merge all these widgets and files into there first but if that doesn't make sense, we can edit these PRs.

## How to test
Navigate to /dashboard and confirm that the statistics appear, that they appear correct, and that they change via the Regional select and the date select

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-168

## Checklists

### Every PR


- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy

- [n/a] Staging smoke test completed

### After merge/deploy

- [n/a] Update JIRA ticket status
